package.json

```diff
@@ -2,12 +2,18 @@
   "name": "single-spa",
   "version": "2.1.8",
   "description": "Multiple applications, one page",
-  "main": "lib/single-spa.js",
+  "main": "dist/single-spa.js",
```

Comment:
the dist file is now bigger (60kb) because it has no dependencies and has async/await runtime built in. It is now script taggable, require-able, and systemjs-able